### DISCUSSION TITLE
Request: Masked DB compatible with 75bp read length

### DISCUSSION THREAD

**Discussion by funnell** [bhattlab/phanta#36] [Phanta DBs] [answered]
Hello,

Would it be possible to have the masked db work with 75bp reads?
Thank you very much!

**Comment by meenachakra** [comment, marked as answer]
Yes! Here you go :)

[Masked database - 75mers.zip](https://github.com/bhattlab/phanta/files/12458797/Masked.database.-.75mers.zip)

**Reply by funnell** [reply]
Thank you very much!

**Comment by meenachakra** [comment]
Please add the file inside that zip to your downloaded masked DB and it should work.
